contracts/Gateway.sol

```diff
@@ -203,8 +203,10 @@ contract Gateway is IGateway, GatewaySettingManager, PausableUpgradeable {
 		require(!order[_orderId].isRefunded, 'OrderRefunded');
 		require(order[_orderId].protocolFee >= _fee, 'FeeExceedsProtocolFee');
 
-		// transfer refund fee to the treasury
-		IERC20(order[_orderId].token).transfer(treasuryAddress, _fee);
+		if (_fee > 0) {
```

Comment:
we should use `_fee != 0`

it's cheaper in gas costs

Reply:
I kindly disagree with this sir.
![code](https://github.com/user-attachments/assets/c3e92a7f-9464-4ae6-af58-d7c17837111a)